server/msteams/client.go

```diff
@@ -1013,6 +1015,12 @@ func (tc *ClientImpl) ListUsers() ([]User, error) {
 			DisplayName: *u.DisplayName,
 			ID:          *u.ID,
 		}
+
+		if u.Mail != nil {
+			users[i].Mail = strings.ToLower(*u.Mail)
+		} else {
```

Comment:
I would check here `u.UserPrincipalName != nil` to avoid to panic in case there is no `UserPrincipalName`